docs/split_keyboards.md

```diff
@@ -35,38 +49,71 @@ split = Split(
     split_flip=True,  # If both halves are the same, but flipped, set this True
     split_side=None,  # Sets if this is to SplitSide.LEFT or SplitSide.RIGHT, or use EE hands
     split_type=SplitType.UART,  # Defaults to UART
-    split_target_left=True,  # If you want the right to be the target, change this to False
+    split_target_left=True,  # Assumes that left will be the one on USB. Set to False if it will be the right
     uart_interval=20,  # Sets the uarts delay. Lower numbers draw more power
     data_pin=None,  # The primary data pin to talk to the secondary device with
     data_pin2=None,  # Second uart pin to allow 2 way communication
-    target_left=True,  # Assumes that left will be the one on USB. Set to False if it will be the right
     uart_flip=True,  # Reverses the RX and TX pins if both are provided
     use_pio=False,  # Use RP2040 PIO implementation of UART. Required if you want to use other pins than RX/TX
 )
 
 ```
 
-### EE HANDS
-If you want to plug USB in on either side, or are using Bluetooth, this is for
-you.
+#### `split_side`
+This tells your micro wich side it handles. For simple case, it is not needed as `split_side = None` will imply:
+- auto dectection of the side from the drive name (ending with 'R'/'L').
+- The `split_target` will be overriden. Each side will act as a `split_target` if connected to a usb host.
 
-Rename your CIRCUITPY drive to something different. The left side must
-end in L, the right must is in R. The name must be 11 characters or less! This is
-a limitation of the filesystem. You will receive an error if you choose a name
-longer than 11 characters. Instructions on how to do that are
-[here](https://learn.adafruit.com/welcome-to-circuitpython/renaming-circuitpy).
-For example on NYQUISTL for left and NYQUISTR for the right.
+This is a sane default settings.
+
+But you still can choose to set all that manually. Some reasons would be:
+- easy both side debug and upload by having both halves connected to usb (and uart between them). Explicit `split_side` and only one `split_target` is needed to avoid each half acting as a `split_target` (duplicating each keypress).
+- wanting to be explicit about hardware discrepancy between halves (others than just reversing the rows see `split_flip`):
 
-For wired connections you don't need to pass anything. For Bluetooth, remove the `split_side` like this
 ```python
-# Wired
-split = Split()
-# Wireless
-split = Split(split_type=SplitType.BLE)
+from storage import getmount
+
+side = SplitSide.RIGHT if str(getmount('/').label)[-1] == 'R' else SplitSide.LEFT
+
+if side == SplitSide.RIGHT:
+    keyboard.col_pins = ...
```

Comment:
This is only needed if PCB's are not interchangeable between sides, which is the common practice. maybe note that this is optional if they have different pinouts.